Dockerfile.panda

```diff
@@ -46,7 +47,8 @@ RUN pyenv install 3.7.3 && \
     pyenv global 3.7.3 && \
     pyenv rehash && \
     pip install --no-cache-dir --upgrade pip==18.0 && \
-    pip install --no-cache-dir -r /tmp/requirements.txt
+    pip install --no-cache-dir -r /tmp/requirements.txt && \
+    pip install pre-commit==2.4.0 pylint==2.5.2
```

Comment:
--no-cache-dir, probably doesn't make a big different but good practice

Reply:
Actually, why don't they just get added to the requirements?

Reply:
I’ll add them to the requirements!